tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/UpdateSolutionCapabilitiesTests.cs

```diff
@@ -32,6 +32,10 @@ public async Task ShouldUpdateCapabilitiesAsync()
         {
             var capabilityRefs = new HashSet<string>(){"C1", "C2"};
 
+            _context.MockSolutionCapabilityRepository.Setup(c =>
+                    c.GetMatchingCapabilitiesCountAsync(capabilityRefs, It.IsAny<CancellationToken>()))
+                .ReturnsAsync(2);
```

Comment:
Could we constant the 2 to give a little meaning to it e.g. expectedCapabilityCount